src/main/kotlin/no/nav/syfo/client/dokarkiv/DokarkivClient.kt

```diff
@@ -113,7 +113,7 @@ class DokarkivClient(
             else -> {
                 log.error("Call to Dokarkiv failed with status: ${response.status}, : ${response.bodyAsText()}")
                 COUNT_METRIKK_FOLLOWUP_LPS_LPS_JOURNALFORT_TIL_GOSYS_FALSE.increment()
-                throw RuntimeException("Failed to call dokarkiv")
+                throw RuntimeException("Failed to call back system with status: ${response.status}, : ${response.bodyAsText()}")
```

Comment:
Vil denne informasjonen nå bli tilgjengeliggjort ut mot konsument?

Reply:
Det vil jeg tro ja. 
Jeg var litt i tvil faktisk, om jeg bør vise : ${response.status}, : ${response.bodyAsText()} 🤔 
Kanskje vi bør fjerne de? 
Jeg var litt ivrig å vise noe forklarende feilmelding, men samtidig kanskje vi bør ikke vise for mye. 
Tenker du at vi kan fjerne de, og bare si: `Failed to call archive system` ?

Reply:
Hvis rotårsaken til feilen er at personen ikke finnes i Pdl, så bør vi kanskje heller først gjøre en sjekk mot PDL, og så gi en fornuftig feilmelding om det.

Reply:
@kaja-nav jeg tror her, vi har allerede gjort en sjekk mot PDL før vi kaller dokarkiv. 
Den blir gjort når vi kaller `pdf = opPdfGenClient.getLpsPdf(followUpPlanDTO)` i `FollowUpPlanSendingService`. 
Eller er det noe annet sjekk du mener?

Reply:
Det ser ut som det bare blir gjort hvis `shouldSendToGeneralPractitioner || shouldSendToNav`. Vi burde kanskje gjøre det uansett?

Reply:
Og så kan det muligens se ut som den koden du refererer til ikke vil feile hvis personen ikke finnes i pdl? At det bare blir "null" i pdfen?

Reply:
Men deretter gjør vi en sjekk for `pdf` og vi kaller ikke dokarkiv dersom den er `null`.

Reply:
Sånn jeg leser det så vil ikke selve pdf'en være null. Pdf'en inneholder navn og adresse fra pdl. Disse vil antakeligvis være null.

Reply:
Jeg tenker at all informasjon som går ut eksternt i feilmeldinger også bør være noe nyttig for konsumentene. F.eks tenker jeg at vi ikke bør si noe om archive system, det blir for internt og blir forvirrende for LPS-leverandører.